app/views/shared/_email_subscribe_unsubscribe_flash.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if @account_flash.include?("email-subscription-success") %>
+
+  <%= render "govuk_publishing_components/components/success_alert", {
+    message: sanitize("You&rsquo;ve subscribed to emails about this page.")  ,
```

Comment:
I wonder if the message and description should be in a locale file?

Reply:
Sure, I'll move them over. 

Out of interest, do we have any sort of style guide / guidance that says when page content should be in a locale file and when it's fine to be in the template? I've seen some templates where everything is in the locale files and some where it's a mixture.

Reply:
Here's how it looks now: 
![image](https://user-images.githubusercontent.com/6362602/141279243-eebe23a6-5e1b-4e1e-8023-179a9f18ddea.png)

Also, moving to locale files has sorted out all the unicode errors I was seeing before so the content is much more readable now

Reply:
Alex to be honest I'm not too sure there is much specific guidance around localisation on GOVUK unfortunately (I might be wrong about that though, I can do some digging). 
I tend to add to locale files anything that could potentially be translated in the future, and/or bits of text that are shared between different partials in the application.